components/Players/VideoPlayer.vue

```diff
@@ -47,7 +48,10 @@ export default Vue.extend({
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       player: null as any,
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ui: null as any
+      ui: null as any,
+      throttledOnVideoProgress: null as DebouncedFunc<
```

Comment:
Why not define a method instead of this?

Reply:
As we discussed, the method way cause the error about `this`. So I used this which is error-free. But the methods work and I can disable the errors

Reply:
@mrtimscampi I thought the same at first but I checked the branch out and I discovered that the problem is not exactly with this because we could always do: ``throttle(this.updateTime, 500);``

However, it's not a problem about this or anything, it's a matter that throttle returns a function, which is the one we need to run, not ``throttle`` itself.